src/components/List.js

```diff
@@ -36,29 +45,47 @@ const List = ({ token }) => {
     <div>
       <h1>List</h1>
 
-      {docs.length === 0 && (
+      {docs.length === 0 ? (
```

Comment:
We changed this to a ternary so that you would either see the prompt to add your first item or the search bar for filtering if a list exists.